StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkSignupPane/PhoneCountryCodePickerView.swift

```diff
@@ -45,10 +45,11 @@ final class PhoneCountryCodePickerView: UIView, UIPickerViewDelegate, UIPickerVi
     init(defaultCountryCode: String?) {
         let locale = Locale.current
         let rowItems = CreateCountryCodeRowItems(locale: locale)
-        let defaultCountryCode = defaultCountryCode ?? locale.stp_regionCode ?? ""
+        let defaultCountryCode = defaultCountryCode ?? locale.stp_regionCode ?? "US"
```

Comment:
Financial Connections is available only in "US" so rather than having no default country code, "US" is more appropriate

that being said, its mostly likely nearly impossible that it even reaches "US" part of the code